packages/user-interface/index.js

```diff
@@ -1,34 +1,48 @@
+    success:
+      '<svg style="pointer-events: none;" class="octicon" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16" width="16" height="16"><path fill-rule="evenodd" d="M13.78 4.22a.75.75 0 010 1.06l-7.25 7.25a.75.75 0 01-1.06 0L2.22 9.28a.75.75 0 011.06-1.06L6 10.94l6.72-6.72a.75.75 0 011.06 0z"></path></svg>',
+    warn:
+      '<svg style="pointer-events: none;" class="octicon" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16" width="16" height="16"><path fill-rule="evenodd" d="M8.22 1.754a.25.25 0 00-.44 0L1.698 13.132a.25.25 0 00.22.368h12.164a.25.25 0 00.22-.368L8.22 1.754zm-1.763-.707c.659-1.234 2.427-1.234 3.086 0l6.082 11.378A1.75 1.75 0 0114.082 15H1.918a1.75 1.75 0 01-1.543-2.575L6.457 1.047zM9 11a1 1 0 11-2 0 1 1 0 012 0zm-.25-5.25a.75.75 0 00-1.5 0v2.5a.75.75 0 001.5 0v-2.5z"></path></svg>',
+    error:
+      '<svg style="pointer-events: none;" class="octicon" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16" width="16" height="16"><path fill-rule="evenodd" d="M1.5 8a6.5 6.5 0 0110.535-5.096l-9.131 9.131A6.472 6.472 0 011.5 8zm2.465 5.096a6.5 6.5 0 009.131-9.131l-9.131 9.131zM8 0a8 8 0 100 16A8 8 0 008 0z"></path></svg>',
   };
 
   const el = document.createElement('div');
-  el.innerHTML = `<div class="js-octolinker-flash flash flash-full ${typeClass[type]}">
-    <div class="container">
-      <button class="octolinker-flash-close flash-close js-flash-close js-flash-close-${id}" type="button" aria-label="Dismiss this message">
-        <svg aria-hidden="true" class="octicon octicon-x" height="16" version="1.1" viewBox="0 0 12 16" width="12"><path d="M7.48 8l3.75 3.75-1.48 1.48L6 9.48l-3.75 3.75-1.48-1.48L4.52 8 .77 4.25l1.48-1.48L6 6.52l3.75-3.75 1.48 1.48z"></path></svg>
+  el.innerHTML = `<div class="position-fixed bottom-0 left-0 ml-5 mb-5 anim-fade-in fast js-octolinker-toast octolinker-toast Toast ${typeClass[type]}" role="log" style="z-index: 101;">
+    <span class="Toast-icon">
+        ${icons[type]}
+    </span>
+    <span class="Toast-content">
+      <div class="octo-toast-image"></div>
+      ${body}
+    </span>
+      <button class="Toast-dismissButton js-octolinker-toast-close-${id}" type="button" aria-label="Close">
+        <svg class="octicon octicon-x" viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M3.72 3.72a.75.75 0 011.06 0L8 6.94l3.22-3.22a.75.75 0 111.06 1.06L9.06 8l3.22 3.22a.75.75 0 11-1.06 1.06L8 9.06l-3.22 3.22a.75.75 0 01-1.06-1.06L6.94 8 3.72 4.78a.75.75 0 010-1.06z"></path></svg>
       </button>
-       <div class="octolinker-flash-image"></div>
-       <div class="octolinker-flash-content">
-        ${body}
-       </div>
-    </div>
   </div>`;
 
-  document.getElementById('js-flash-container').append(el);
-
-  return el;
-};
-
-export const removeAllNotifications = () => {
-  document
-    .querySelectorAll('.js-octolinker-flash')
-    .forEach((el) => el.remove());
+  document.body.append(el);
```

Comment:
We could insert these right into the alert container GitHub uses:

```js
document.querySelector('#js-flash-container').append(...el.children);
```

This element should exist on every page, I haven't seen one without it.

They do some special handling of alerts but I'm not really sure what it's for. It might be to swap them out if the pjax response is an alert, in which case it might not be a good idea to put them in this container. You can see this in the `github-bootstrap.js` file by searching for `js-flash-container'`.

Reply:
The reason why I moved away from this `#js-flash-container` container was to ensure the toast keeps showing even GitHub removes this container one day. To be fair, OctoLinker is using this container since a couple of years, but you never know. Also, as far as I tested this does not cause any issues when switching between pjax pages. I thinks it's best to rely as little as possible on the GitHub DOM. That said, in this PR I use quite a few CSS classes from GitHub which can change as well 🤷‍♂️ 

Unless you have strong feelings about this container, I tend to keep it as it is.

Reply:
I just checked and GitHub is appending toast messages to the body as well.
![image](https://user-images.githubusercontent.com/1393946/89132958-3ac2d580-d518-11ea-884d-637a8aca9a61.png)

Reply:
I thought they were adding them to the alert container but I guess not. As long as they work leaving them out is the better way since their JS looks to clear the alerts if a new one is sent via pjax.